src/olympia/activity/utils.py

```diff
@@ -195,3 +218,20 @@ def filter_queryset_to_pending_replies(queryset, log_type_ids=NOT_PENDING_IDS):
     if not latest_reply:
         return queryset
     return queryset.filter(created__gt=latest_reply.created)
+
+
+def bounce_mail(message, reason):
+    if (not type(message) is dict or
+            ('From' not in message and 'ReplyTo' not in message)):
```

Comment:
if `message` isn't a `dict` then it might not have a `.get()` method, that was my thinking.